ECHO-COVID19.ipynb

```diff
@@ -164,24 +390,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "latest = data[(data[\"SCHEDULE_DATE\"] >= '2020-03-01') & (data[\"SCHEDULE_DATE\"] < '2020-04-01')] #filter to March\n",
+    "latest = data[(data[\"SCHEDULE_DATE\"] >= '2020-04-01') & (data[\"SCHEDULE_DATE\"] < '2020-05-01')] #filter to March\n",
     "\n",
     "if (len(latest.index)>0):\n",
     "    #get facility information from ECHO\n",
     "    sql = \"select FAC_NAME, NPDES_IDS, FAC_LAT, FAC_LONG, CWA_QTRS_WITH_NC\" + \\\n",
-    "        \" from ECHO_EXPORTER \" + \\\n",
-    "        \" where NPDES_FLAG = 'Y' and FAC_STATE = '\" + my_state + \"'\"\n",
+    "        \" from ECHO_EXPORTER where NPDES_FLAG = 'Y' \"\n",
     "    url='http://apps.tlt.stonybrook.edu/echoepa/?query='\n",
     "    data_location=url+urllib.parse.quote(sql)\n",
     "    echo_data = pd.read_csv(data_location,encoding='iso-8859-1',header = 0)\n",
     "    echo_data.set_index( 'NPDES_IDS', inplace=True )\n",
-    "\n",
+    "    \n",
     "    #merge echo and npdes data\n",
     "    latest = latest.join(echo_data)\n",
     "    print(latest)\n",
     "    \n",
     "else:\n",
-    "    print(\"Actually, there were no reporting violations for March in \"+my_state)\n",
+    "    print(\"Actually, there were no reporting violations for April\")\n",
```

Comment:
This is good (though, better if it's programmatic) but the map in the next cell fails with error if there's no data

Reply:
Yea, I had it set to go through March data because it was too soon for April data. I will double check and fix this.

Reply:
Added logic to prevent the map cell from running if no data